### ISSUE TITLE
Node height dimension incorrect with custom node template

### ISSUE THREAD

**Issue by bohdyone** [swimlane/ngx-graph#162]
**I'm submitting a ...**  (check one with "x")
```
[ x ] bug report => search github for a similar issue or PR before submitting
[   ] feature request
[   ] support request => Please do not submit support request here
```

**Current behavior**
Node height dimension is unreliable with multi-line custom node template. This also seems to affect layout.

**Expected behavior**
 Custom node template should be able to be filled correctly and also render without occlusion.

**Reproduction of the problem**
https://stackblitz.com/edit/angular-ucydej

**What is the motivation / use case for changing the behavior?**
To allow for displaying of detailed information within the node body, allowing the resulting graph to be exported and printed, unlike when using tooltips.

**Please tell us about your environment:**

* **ngx-graph version:** 6.0.0-rc1

* **Angular version:** 7.2.3

* **Browser:** [ Chrome ]

* **Language:** [ all ]

**Comment by assafsun** [comment]
Which ngx-graph version? is this the code inside master?

ngx-graph supports setting different nodes sizes in order to allow a better layout from dagre.
can this help?

**Comment by bohdyone** [comment]
Hi @assafsun, please check the linked stackblitz.
* **ngx-graph version:** 6.0.0-rc1 (from npm)

I would expect the node height to be calculated automatically based off the svg rect rendered in the template - but this doesn't appear to be working in version 6. It was ok in version 5.

**Comment by marjan-georgiev** [comment]
This should be fixed in version 6.0.0-rc.3

Here's the updated stackblitz: https://stackblitz.com/edit/angular-edligh?file=src%2Fapp%2Fapp.module.ts
